examples/step-3/doc/results.dox

```diff
@@ -124,6 +124,31 @@ suggestions:
   potentially non-zero Neumann conditions). You will see this if you
   run the program.
 
+  An alternative way to change the boundary indicator is to label
+  the boundaries by the cartesian coordinates of the cell centers.
+  For example, we can label all of the cells along the top and
+  bottom boundaries with a boundary indicator 1 by checking to
+  see if the cell centers' y-coordiantes are within a tolerance
+  (here 1e-12) of -1 and 1. Try this immediately after calling
+  <code>GridGenerator::hyper_cube</code>, as before:
+  @code
+  for (; cell != endc; ++cell)
```

Comment:
Can't you use a range-based for loop?

Reply:
What do you mean by range-based for loop?

Reply:
If you have a look at `Step3::assemble_system`, you will see that the loop over the cells looks like:
```
for (const auto &cell : dof_handler.active_cell_iterators())
{
  ...
}
```
We generally prefer this  syntax over
```
for (; cell != endc; ++cell)
{
  ...
}
```

Reply:
I see, thanks for the suggestion! Just made the change.